backend/config/config.go

```diff
@@ -129,6 +133,7 @@ func DefaultConfig() *Config {
 		NomadReadOnly:    false,
 		NomadAddress:     "http://127.0.0.1:4646",
 		NomadHideEnvData: false,
+                NomadDisableFS:   false,
```

Comment:
This doesn't need to be `false` since Go defaults to `false` but leaving the decision on @jippi since everything else has been initialized like that

Reply:
Also please run `go fmt` on the code.